solidity/contracts/token/libs/MovableCollateralRouter.sol

```diff
@@ -7,18 +7,20 @@ import {TokenRouter} from "./TokenRouter.sol";
 import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import {Router} from "../../client/Router.sol";
+import {Quotes} from "../../libs/Quotes.sol";
 
 struct MovableCollateralRouterStorage {
-    mapping(uint32 routerDomain => bytes32 recipient) allowedRecipient;
-    mapping(uint32 routerDomain => EnumerableSet.AddressSet bridges) allowedBridges;
-    EnumerableSet.AddressSet allowedRebalancers;
+    mapping(uint32 routerDomain => bytes32 recipient) recipient;
+    mapping(uint32 routerDomain => EnumerableSet.AddressSet bridges) bridges;
+    EnumerableSet.AddressSet rebalancers;
 }
 
 abstract contract MovableCollateralRouter is TokenRouter {
     using SafeERC20 for IERC20;
     using EnumerableSet for EnumerableSet.AddressSet;
+    using Quotes for Quote[];
 
-    MovableCollateralRouterStorage private $;
+    MovableCollateralRouterStorage private allowed;
```

Comment:
Somewhat related, but how does this show up with the built in "getter" view calls for clients like ethers or viem?

Reply:
its private, so its not exposed at all
the getters are built from the public functions below which is consistent with the old interface